doc/source/run.rst

```diff
@@ -0,0 +1,38 @@
+Running Tenks
+=============
+
+Commands
+--------
+
+Tenks has a variable ``cmd`` which specifies the command to be run. This
+variable can be set in your override file (see :ref:`configuration`). The
+possible values it can take are:
+
+* ``deploy``: create a virtual cluster to the specification given. This is the
+  default command.
+* ``teardown``: tear down any existing virtual cluster with the specification
+  given.
+
+Execution
+---------
+
+Currently, Tenks does not have a CLI or wrapper. Deployment can be run by
+calling ``ansible-playbook --inventory ansible/inventory ansible/deploy.yml
```

Comment:
Again, a separate line would be clearer and easier to copy/pasta.

Reply:
Yep.